src/cli/pack_command.zig

```diff
@@ -535,34 +535,77 @@ pub const PackCommand = struct {
         while (iter.next().unwrap() catch null) |entry| {
```

Comment:
Let's change this function to instead loop through the `bundled_deps` list, try to open each, and send the dir through `addBundledDep` if it exists, like you suggested. This way we can avoid opening both directories of a scoped package